packages/babel-parser/test/fixtures/typescript/class/static-static/input.ts

```diff
@@ -0,0 +1,6 @@
+class C {
+  static
+  static
+  static
+  a() {}
+}
```

Comment:
Can you add a new test case?
```ts
class C {
  protected static
  static a() {}
}
```
This test is probably also fixed in this PR.

Reply:
This test case will produce a compiler error, mimicking TypeScript's behavior (tsc). 
If we analyze the transformation steps, removing the protected modifier would first result in the following code structure:

```JavaScript
class C {
            static
  static a() {}
}
```

This modified code will then generate the following syntax error:

```
Uncaught SyntaxError: Unexpected identifier 'a'
```

Reply:
Yes. Previously we throw duplicate `static` modifier, in this PR we will align to the tsc behavior.